toolkit/src/components/cut/filter-bar/search.hbs

```diff
@@ -0,0 +1,26 @@
+{{!
+  Copyright (c) HashiCorp, Inc.
+}}
+
+{{#if @group}}
+  {{#let @group as |SG|}}
+    <SG.TextInput
+      @type='search'
+      placeholder='Search'
+      aria-label='Search'
+      {{on 'keyup' @onSearchKeyup}}
+      {{on 'input' @onSearchInput}}
```

Comment:
not sure, but maybe focus lost is a good one to handle too

Reply:
Same as below re: as designed. Passing on the feedback to design.